.gitignore

```diff
@@ -9,3 +9,5 @@ pom.xml.asc
 *.class
 .lein-*
 .nrepl-port
+.clj-kondo/*
```

Comment:
This can simply be `.cache` and it will exclude the cache folder for clj-kondo and clojure-lsp

Reply:
When running the steps for importing dependency clj-kondo configs, clj-kondo places hiccup's own config in this directory. This means `.clj-kondo/hiccup` would need to be ignored as well, making for a total of two lines again.

Because this can be tedious with more dependencies, I used this whitelist approach on the project I copied these .gitignore lines from. I think this is conceptually the nicer approach (less likely to break in the future), though it doesn't really matter in this specific instance.

Reply:
Alright